### PR TITLE
Don't store axes labels in plot config

### LINKED ISSUE #938: Replace "Concentration (molar)" and "Concentration (mass)" by "Concentration" in plot axis labels
https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/928#issuecomment-1124755911

### LINKED ISSUE #939: `DefaultPlotConfiguration` should not store axis captions created from dimension name
When the user does not explicitely define axis lables and these are generated from dimension name, the caption should be not stored. Otherwise, when chaning units and re-using the configuration in a new plot, the old caption is shown.

Create a plot with x-axis units "week":
```
# import observed data (will return a list of DataSet objects)
dataSet <- loadDataSetsFromExcel(
  xlsFilePath = getTestDataFilePath("CompiledDataSetStevens2012.xlsx"),
  importerConfiguration = loadDataImporterConfiguration(getTestDataFilePath("ImporterConfiguration.xml"))
)
# observed only
myCombDat <- DataCombined$new()
myCombDat$addDataSets(dataSet)

plotConfiguration <- ospsuite::DefaultPlotConfiguration$new()
plotConfiguration$xUnit <- ospUnits$Time$`week(s)`
plotIndividualTimeProfile(myCombDat, plotConfiguration)
```

image

Change the units to minutes and plot again - old caption is shown

```
plotConfiguration$xUnit <- "min"
plotIndividualTimeProfile(myCombDat, plotConfiguration)
```

image

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Closes #938
Closes #939

### CODE REVIEW

**Review by PavelBal — APPROVED** [review]

**Comment by msevestre** [comment]
@PavelBal  please give me a chance to review changes before merging.

**Comment by PavelBal** [comment]
Ok, sorry.

File: R/plot-individual-time-profile.R [hunk 1]
```diff
@@ -55,6 +54,13 @@ plotIndividualTimeProfile <- function(dataCombined,
     generalPlotConfiguration = defaultPlotConfiguration
   )
 
+  # axes labels -----------------------------
+
+  plotType <- class(timeProfilePlotConfiguration)[[1]]
```
Comment (msevestre) [inline R/plot-individual-time-profile.R:59]: What does this do here?

File: R/utilities-plotting.R [hunk 2]
```diff
@@ -73,6 +73,80 @@
...
+  yDimensionString <- unique(data$yDimension)
+
+  # Currently, hard code any of the different concentration dimensions to just
+  # one dimension: "Concentration"
+  #
+  # https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/938
+  concDimensions <- c(
+    ospDimensions$`Concentration (mass)`,
+    ospDimensions$`Concentration (molar)`,
+    ospDimensions$`Concentration (molar) per time`
```
Comment (msevestre) [inline R/utilities-plotting.R:106]: This is NOT concentration. This is concentration per time

**Review by msevestre — COMMENTED** [review]
I think the concentration per time mapped as concentration needs to be reverted. This is another dimension

**Pull request merged**